src/i18n/de-DE/state_of_css.yml

```diff
@@ -0,0 +1,464 @@
+      t: :nth-last-child()
+    - key: options.tree_document_structure.first_child
+      t: :first-child
+    - key: options.tree_document_structure.last_child
+      t: :last-child
+    - key: options.tree_document_structure.only_child
+      t: :only-child
+    - key: options.tree_document_structure.nth_of_type
+      t: :nth-of-type()
+    - key: options.tree_document_structure.nth_last_of_type
+      t: :nth-last-of-type()
+    - key: options.tree_document_structure.first_of_type
+      t: :first-of-type
+    - key: options.tree_document_structure.last_of_type
+      t: :last-of-type
+    - key: options.tree_document_structure.only_of_type
+      t: :only-of-type
+    - key: options.tree_document_structure.lang
+      t: :lang()
+    - key: options.tree_document_structure.is
+      t: :is()
+    - key: options.tree_document_structure.where
+      t: :where()
+    - key: options.tree_document_structure.has
+      t: :has()
+
+    - key: features_others.attributes
+      t: Attribute
+
+    - key: options.attributes.presence
+      t: div[foo] (Vorhandensein)
+    - key: options.attributes.equality
+      t: div[foo="bar"] (Gleichwertigkeit)
+    - key: options.attributes.starts_with
+      t: div[foo^="bar"] (Beginnt mit)
+    - key: options.attributes.ends_with
+      t: div[foo$="bar"] (Endet mit)
+    - key: options.attributes.contains_word
+      t: div[foo~="bar"] (Beinhaltet Wort)
+    - key: options.attributes.contains_substring
+      t: div[foo*="bar"] (Beinhaltet Zeichen)
+
+    - key: features_others.links_urls
+      t: Links/URLs
+
+    - key: options.links_urls.any_link
+      t: :any-link
+    - key: options.links_urls.link_visited
+      t: :link and :visited
+    - key: options.links_urls.local_link
+      t: :local-link
+    - key: options.links_urls.target
+      t: :target
+
+    - key: features_others.interaction
+      t: Interaktion
+
+    - key: options.interaction.hover
+      t: :hover
+    - key: options.interaction.active
+      t: :active
+    - key: options.interaction.focus
+      t: :focus
+    - key: options.interaction.focus_within
+      t: :focus-within
+    - key: options.interaction.focus_visible
+      t: :focus-visible
+
+    - key: features_others.form_controls
+      t: Formularfelder
+
+    - key: options.form_controls.enabled_disabled
+      t: :enabled and :disabled
+    - key: options.form_controls.read_only_write
+      t: :read-only and :read-write
+    - key: options.form_controls.placeholder_shown
+      t: :placeholder-shown
+    - key: options.form_controls.default
+      t: :default
+    - key: options.form_controls.checked
+      t: :checked
+    - key: options.form_controls.indeterminate
+      t: :indeterminate
+    - key: options.form_controls.valid_invalid
+      t: :valid and :invalid
+    - key: options.form_controls.user_invalid
+      t: :user-invalid
+    - key: options.form_controls.in_out_range
+      t: :in-range and :out-of-range
```

Comment:
@abaldeweg nice work, but don't forget to translate strings with word "and", for example:

>      `t: <code>min()</code>, <code>max()</code>, and <code>clamp()</code>`
>      `t: :in-range and :out-of-range`